code/obj/artifacts/artifactprocs.dm

```diff
@@ -447,6 +447,8 @@
 					src.ArtifactActivated()
 
 /obj/proc/ArtifactTouched(mob/user as mob)
+	if (get_dist(src, user) > 1)
```

Comment:
interact_dist

Reply:
Sorry, I'm a bit new to coding in this game, I couldn't find any interact_dist() proc. I tried !in_interact_range() and !can_reach() instead if you were meaning any of those, but they didn't work for machinery artifacts, just changing that one line.

Reply:
the former yes, should work

Reply:
Just had wrong first argument, updated